### PR TITLE
Content; new war events!

### PR DESCRIPTION
**Pull request by cqxsc**
Just adds in more war-related events!

To both war.json (in the events folder) and also new injury events which would be induced by wars between two Clans.

### CODE REVIEW

File: resources/dicts/events/injury/general/general.json [hunk 1]
```diff
@@ -795,5 +795,114 @@
...
+        "tags": [
+            "Newleaf",
+            "Greenleaf",
+            "Leaf-fall",
+            "Leaf-bare",
+            "other_clan",
+            "rel_down",
+            "war"
+        ],
+        "event_text": "While patrolling the o_c borders, m_c stepped on a concealed trap set by what seemed to be o_c warriors. The snare inflicts a deep wound, causing m_c to limp back to camp, flanked by other cats on the patrol.",
```
Comment (keyraven) [inline resources/dicts/events/injury/general/general.json]: Can cats set snares? In the books, I believe Hawkfrost found a snare set up by humans.  Maybe the trap could be changed to a pit-trap, or something more realistic for a cat to create?

File: resources/dicts/events/injury/general/general.json [hunk 2]
```diff
@@ -795,5 +795,114 @@
...
+            "Leaf-bare",
+            "other_clan",
+            "rel_down",
+            "war",
+            "other_cat",
+            "rc_to_mc",
+            "respect",
+            "trust"
+        ],
+        "event_text": "During a skirmish, m_c bounds into battle against an o_c warrior, protecting r_c from a powerful blow. The aftermath turns up with m_c wincing in pain from the strike r_c was about to take.",
```
Comment (keyraven) [inline resources/dicts/events/injury/general/general.json]: The second sentence here is a bit oddly wordy.  I suggest rewording the second sentence. One possible suggestion: 

"Afterwards, r_c helps m_c limp back home."
"m_c bears the brunt of the strike, and r_c helps them back to camp."

File: resources/dicts/events/injury/general/general.json [hunk 3]
```diff
@@ -795,5 +795,114 @@
...
+            "Greenleaf",
+            "Leaf-fall",
+            "Leaf-bare",
+            "other_clan",
+            "rel_down",
+            "war",
+            "other_cat",
+            "other_cat_kit"
+        ],
+        "event_text": "As o_c attempted a surprise attack on the c_n camp, m_c fiercely guarded the nursery, although it didn't end with {PRONOUN/m_c/object} unscathed, a gushing wound in {PRONOUN/m_c/poss} flank.",
```
Comment (keyraven) [inline resources/dicts/events/injury/general/general.json]: Slightly incorrect grammar here - you can't connect two independent clauses, then two dependent clauses, in a row with commas.  I suggest: 

"While o_c attempted a surprise attack on the c_n camp, m_c fiercely guarded the nursery. However, it didn't end with m_c unscathed. {PRONOUN/m_c/subject/CAP} walked away with a gushing wound in {PRONOUN/m_c/poss} flank."

At first read, it is also unclear who is unscathed - the camp or m_c, so I suggested you replace the pronoun with m_c's name.

File: resources/dicts/events/injury/general/general.json [hunk 4]
```diff
@@ -795,5 +795,114 @@
...
+        "tags": [
+            "Newleaf",
+            "Greenleaf",
+            "Leaf-fall",
+            "Leaf-bare",
+            "other_clan",
+            "rel_down",
+            "war"
+        ],
+        "event_text": "On a border patrol near the o_c border, m_c couldn't take it with all of o_c's jeers and smirks- lashing out at the first cat {PRONOUN/m_c/subject} saw jeering at the c_n cats. The hotheaded move didn't seem quite smart, though, as m_c didn't escape the skirmish without drawing blood.",
```
Comment (keyraven) [inline resources/dicts/events/injury/general/general.json]: The first sentence is wordy, and mixes it's tenses. The second sentence make it unclear who was injured - m_c or the o_c cat. I suggest: 

"On a border patrol near the o_c border, m_c could no longer take o_c's jeers and smirks. {PRONOUN/m_c/subject/CAP} lashed out at the first jeering cat {PRONOUN/m_c/subject} saw. The hotheaded move wasn't smart, though, as m_c couldn't escape the skirmish without injury."

File: resources/dicts/events/war.json [hunk 5]
```diff
@@ -27,19 +38,46 @@
...
-    "With the many injuries both sides have suffered, o_c and c_n found common ground and ended the war."
+    "With the many injuries both sides have suffered, o_c and c_n found common ground and ended the war.",
+    "The dust has finally settled of the war between o_c and c_n.",
+    "The leaders of both o_c and c_n decide that for the wellbeing of both Clans, the war should come to an end now.",
+    "After lots of bloodshed of both o_c and c_n cats, the two Clans have come to terms of peace.",
+    "The battles between o_c and c_n have ceased, the war now finally coming to an end.",
+    "o_c and c_n have had a long and complex past, but it's time for their battles to come to an end.",
+    "Many battles have been fought between o_c and c_n, but it's time for those to stop now.",
+    "The cats of both o_c and c_n have come to a realization that war isn't the best solution, deciding to end it now.",
+    "The defiantly puffed out chests of o_c and c_n have turned into tired, sulking bodies, and the cats are glad as the war finally comes to a close.",
```
Comment (keyraven) [inline resources/dicts/events/war.json]: Although I love those conclusion texts, I find "defiantly puffed out chests" to be slightly awkward. I suggest: 

"flashing fangs"
"defiant war cries" 
"puffed out chests" (remove "defiantly") 

I also making "...the cats are glad as the war finally comes to a close" into it's own sentence for clarity. At the moment, it reads (at first blush) as being part of a list with "tired, sulking bodies".

**Review by keyraven — CHANGES_REQUESTED** [review]

File: resources/dicts/events/injury/general/general.json (hunk 1 above)
Comment (cqxsc) [reply resources/dicts/events/injury/general/general.json]: ohh good idea! maybe cats could use some random debris and stuff to make a pitfall trap
i'll change it to that

File: resources/dicts/events/injury/general/general.json (hunk 3 above)
Comment (cqxsc) [reply resources/dicts/events/injury/general/general.json]: i think i'll take your suggestion to replace it with, seems a bit less wordy and more grammatically correct

File: resources/dicts/events/injury/general/general.json (hunk 4 above)
Comment (cqxsc) [reply resources/dicts/events/injury/general/general.json]: will do

File: resources/dicts/events/injury/general/general.json (hunk 2 above)
Comment (cqxsc) [reply resources/dicts/events/injury/general/general.json]: will do

File: resources/dicts/events/war.json (hunk 5 above)
Comment (cqxsc) [reply resources/dicts/events/war.json]: i think piercing war cries would be good and yeah, wasnt sure with the whole sulking bodies thing to begin with so will do

**Review by keyraven — APPROVED** [review]
